src/commands/math/commands.js

```diff
@@ -369,11 +373,11 @@ var SummationNotation = P(MathCommand, function(_, super_) {
   };
 });
 
-LatexCmds['∑'] =
+LatexCmds['?'] =
 LatexCmds.sum =
 LatexCmds.summation = bind(SummationNotation,'\\sum ','&sum;');
 
-LatexCmds['∏'] =
+LatexCmds['?'] =
```

Comment:
@sclower is it easy for you to find the commit where the byte order mark confusion slipped in? Seems like we'll want to be on the lookout for this issue to avoid more confusion down the road.